completions/bfs.bash

```diff
@@ -0,0 +1,170 @@
+# bash completion script
+
+_bfs ()
+{
+  local cword=$COMP_CWORD
+  local cur="${COMP_WORDS[$cword]}"
+  local prev prev1
+  if (($cword > 0)); then
+    prev="${COMP_WORDS[$(($cword-1))]}"
+  fi
+  if (($cword > 1)); then
+    prev1="${COMP_WORDS[$(($cword-2))]}"
+  fi
+
+  # arguments with a special completion procedure
+  local options_special='-fprintf -D -S -regextype -fstype -gid -uid -group -user -perm -type -xtype'
+
+  # arguments whose values should not be completed
+  # (e.g. because they are numeric, glob, regexp, time, etc.)
+  local options_nocomp='-maxdepth -mindepth -amin -Bmin -cmin -mmin -asince -Bsince -csince -msince -atime -Btime -ctime -mtime -ilname -iname -ipath -iregex -iwholename -inum -links -lname -name -path -wholename -regex -since -size -used -printf -newerat -newerBt -newerct -newermt -xattrname'
+
+  # arguments whose value is a filename
+  local options_filecomp='-anewer -Bnewer -cnewer -mnewer -newer -samefile -fls -fprint -fprint0 -neweraa -neweraB -newerac -neweram -newerBa -newerBB -newerBc -newerBm -newerca -newercB -newercc -newercm -newerma -newermB -newermc -newermm'
+
+  # arguments whose value is a dirname
+  local options_dircomp='-f'
+
+  # options with no value
+  local flags='-H -L -P -E -X -d -s -x -O1 -O2 -O3 -not -and -or -exclude -color -nocolor -daystart -depth -follow -ignore_readdir_race -noignore_readdir_race -mount -nohidden -status -unique -warn -nowarn -xdev -acl -capable -empty -executable -readable -writable -false -true -hidden -nogroup -nouser -sparse -xattr -delete -rm -exit -ls -print -print0 -printx -prune -quit -version -help'
+
+  local all_options="${flags} ${options_special} ${options_nocomp} ${options_filecomp} ${options_dircomp}"
+
+  # completions which require parsing the whole command string
+  case "${COMP_WORDS[*]}" in
+    (*\ -exec\ *|*\ -execdir\ *|*\ -ok\ *|*\ -okdir\ *)
```

Comment:
It would be nice if these could be at least completed from prefixes.  Right now `bfs -exe<TAB>` fills in `bfs -executable` for example.

Reply:
Fixed in 799d521